src/Altinn.Common/Altinn.EFormidlingClient/Altinn.EFormidlingClient/Models/Capabilities.cs

```diff
@@ -0,0 +1,56 @@
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text.Json.Serialization;
+
+namespace Altinn.Common.EFormidlingClient.Models
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="Capabilities"/> class.
+    /// </summary>
+    public class Capabilities
```

Comment:
```suggestion
    public class Capabilities : List<Capabilities>
```
Could you please test if this would work? Removing all other implementation within the class.. 
We would avoid that extra property that doesn't add any extra infromation if we define the class like this.